workers/ohsome_quality_analyst/cli/cli.py

```diff
@@ -82,7 +83,15 @@ def list_fid_fields():
 def get_available_regions():
     """List available regions."""
     regions = asyncio.run(db_client.get_available_regions())
-    click.echo(regions)
+    with open("available_regions.csv", "w", newline="") as csvfile:
+        writer = csv.writer(csvfile)
+        for feature in regions["features"]:
+            writer.writerow([feature["id"]] + [feature["properties"]["name"]])
+        csvfile.close()
+    with open("available_regions.csv", newline="") as read_file:
```

Comment:
You could write and print out information in one loop. Also since the information is already present in memory (varibale `regions`) there is no need to read the csv file again.